docs/documentation/server_admin/topics/organizations/managing-members.adoc

```diff
@@ -0,0 +1,9 @@
+[id="managing-organization-members_{context}"]
+
+=  Managing Members
+[role="_abstract"]
+Manage Members
+
+== Adding an existing realm user as a member
+== Understanding Managed and Unmanaged Members
```

Comment:
```suggestion
== Managed and unmanaged members
```

Reply:
If this a concept, not a task, we use nouns not verbs.

Reply:
It is a concept.